tests/operators/test_operator_norm.py

```diff
@@ -0,0 +1,96 @@
+"""Tests for computing the operator norm for linear operators."""
+
+# Copyright 2024 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from math import cos
+from math import pi
+from math import prod
+
+import torch
+from mrpro.operators import EinsumOp
+from mrpro.operators import FastFourierOp
+
+from tests import RandomGenerator
+
+
+def test_correct_result():
```

Comment:
test_operatornorm_result?

Reply:
split this into two tests

Reply:
just out of curiosity, why is it better to split it?

Reply:
these are two completely different tests, nothing is shared between the two blocks. no reason to have them as one test.